### PR TITLE
styling for banner info

### PR DESCRIPTION
**Pull request by thegreatgulper**
I have removed styling from <p> and enforced padding around the banner

### CODE REVIEW

File: src/wmnds/patterns/banner/_banner.njk [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 <div class="wmnds-grid wmnds-banner-container">
-   <div class="wmnds-col-auto wmnds-float-left wmnds-m-r-sm">
+   <div class="wmnds-col-auto wmnds-float-left ">
       {{
          wmndsPhaseIndicator() | indent(6) | trim
       }}
    </div>
    <div class="wmnds-col-auto">
-      <p>This is a new service - your <span class="wmnds-banner-container__feedback">feedback</span> will help us to improve it.</p>
+      <p>This is a new service - your <a href="#" class="wmnds-banner-container__feedback">feedback</a> will help us to improve it.</p>
```
Comment (daylesalmon) [inline src/wmnds/patterns/banner/_banner.njk:10]: As the p has been styled, could you give it a BEM class such as
```.wmnds-banner-container__text```

Thanks!

File: src/wmnds/patterns/banner/_banner.scss [hunk 2]
```diff
@@ -1,7 +1,8 @@
 .wmnds-banner-container {
   border-bottom: 1px solid get-color(text);
-
-  &__feedback {
-    border-bottom: 1px solid get-color(text);
+  padding: $size-md 0;
+  p {
```
Comment (daylesalmon) [inline src/wmnds/patterns/banner/_banner.scss:4]: Same again with the BEM class here.

File: src/wmnds/patterns/banner/_banner.njk (hunk 1 above)
Comment (thegreatgulper) [reply src/wmnds/patterns/banner/_banner.njk]: Cool, make sense, added

File: src/wmnds/patterns/banner/_banner.scss (hunk 2 above)
Comment (thegreatgulper) [reply src/wmnds/patterns/banner/_banner.scss]: same here

**Pull request merged**